src/execution/operator/csv_scanner/util/csv_error.cpp

```diff
@@ -67,6 +68,18 @@ void CSVErrorHandler::ErrorIfNeeded() {
 	}
 }
 
+void CSVErrorHandler::ErrorIfAny() {
```

Comment:
Is this method used?

If so, is there a guarantee that the errors here need to get a line for them (not that not all errors need line numbers) or that the prior threads will be done when this is called?

Reply:
It's used in `CSVGlobalState::FinishFile`. The reason I added this was that this used to call `ErrorIfNeeded()` - which would silently not error despite errors being present if not all line info was added (which was a bug). I made this an explicit bug now at this stage (as opposed to just swallowing the errors).